src/main/java/org/peergos/util/InstanceAdmin.java

```diff
@@ -0,0 +1,44 @@
+package org.peergos.util;
+
+import org.peergos.cbor.CborObject;
+import org.peergos.cbor.Cborable;
+
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.concurrent.CompletableFuture;
+
+
+public interface InstanceAdmin {
+
+    CompletableFuture<VersionInfo> getVersionInfo();
```

Comment:
Why do we need this class? What does kubo do for version, is it json?

Reply:
removed class